charts/argo-cd/templates/argocd-repo-server/serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.repoServer.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "argo-cd.repoServerServiceAccountName" . }}
+  labels:
```

Comment:
Could you please add the option to set annotations on the service account. This is required for things like workload identity on Google Cloud and AWS and would be very helpful for any reposerver that needs to pull info from S3 or google storage.